src/main/java/org/euvsvirus/cac/controller/api/TeamController.java

```diff
@@ -45,12 +45,10 @@ public TeamController(CacUserTeamService cacUserTeamService, CacUserService cacU
         return cacUserTeamService.getTeamSkills(skillFilter);
     }
 
-    /**
-     * TODO: Only allow access to the own team.
-     */
     @GetMapping(value = "/members", produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<List<User>> getUsersForTeam(@RequestParam String teamId) {
-        return new ResponseEntity<>(cacUserTeamService.findUsersByTeamId(teamId), HttpStatus.OK);
+        User currentUser = CacUserService.getCurrentUser();
```

Comment:
Würde ich in den Service packen, dann kann man das da auch besser testen... also sobald wir Tests schreiben 🤷